README.md

```diff
@@ -1,2 +1,48 @@
-# BOCS Control
-Control scripts for the BOCS.
+# BOCS Data Collection
+
+Contains the source code for the `python3-bocs` Debian package that runs the BOCS air quality instrument, recording data to local storage.
+Released Deb files are available at <TODO>.
```

Comment:
Do we need to update this to state that the latest released Debs are available through GitHub releases?

Reply:
I'll do this once the release has been finalised and we have a URL to the Deb